Source/Core/Core/PowerPC/Jit64/RegCache/JitRegCache.h

```diff
@@ -42,6 +42,7 @@ class RCOpArg
   Gen::OpArg Location() const;
   operator Gen::OpArg() const& { return Location(); }
   operator Gen::OpArg() const&& = delete;
+  operator bool() const;
```

Comment:
```suggestion
  explicit operator bool() const;
```

is generally more desirable, since you don't get wonky conversions to bool where they're usually unexpected